src/components/TopBar.js

```diff
@@ -49,6 +47,7 @@ export default function TopBar({ menuAction }) {
         </div>
 
         <Typography className={classes.title}>
+          <Button variant="contained" color="primary">Custom Event</Button>
```

Comment:
I think the verbiage should be "Add Custom Event" instead of just "Custom Event." I also think it would be good to have some x-axis padding, as it seems too close to the term display.